### PR TITLE
feat(fines): added statistics for fines

### PR DESCRIPTION
**Pull request by Her0elt**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

Issue number: closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] The fixtures have been updated if needed (for migrations)
- [ ] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [ ] Format and lint (`make format` and `make flake8`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: app/group/serializers/fine.py [hunk 1]
```diff
@@ -114,3 +114,24 @@ class Meta:
...
+        fields = ("payed", "approved_and_not_payed", "not_approved")
+
+    def get_payed(self, obj):
+        return obj.fines.filter(payed=True).count()
+
+    def get_approved_and_not_payed(self, obj):
+        return obj.fines.filter(payed=False, approved=True).count()
+
+    def get_not_approved(self, obj):
+        return obj.fines.filter(approved=False).count()
```
Comment (olros) [inline app/group/serializers/fine.py:137]: Dette blir feil. `amount` i hver bot må summeres opp. Altså skal én betalt bot med `amount: 4` bli til **4**.

**Review by olros — CHANGES_REQUESTED** [review]

File: app/group/serializers/fine.py [hunk 2]
```diff
@@ -114,3 +115,28 @@ class Meta:
...
+        model = Group
+
+        fields = ("payed", "approved_and_not_payed", "not_approved")
+
+    def get_sum(self, obj, kwargs):
+        sum = obj.fines.filter(**kwargs).aggregate(sum=Sum("amount"))["sum"]
+        return sum if sum else 0
+
+    def get_payed(self, obj):
+        return self.get_sum(obj, {"payed": True})
```
Comment (olros) [inline app/group/serializers/fine.py:136]: Funker ikke dette også? Mer clean syns jeg kanskje isåfall

```suggestion
    def get_sum(self, obj, *args, **kwargs):
        sum = obj.fines.filter(*args, **kwargs).aggregate(sum=Sum("amount"))["sum"]
        return sum if sum else 0

    def get_payed(self, obj):
        return self.get_sum(obj, payed=True)
```

Comment (Her0elt) [reply app/group/serializers/fine.py]: alt er lov i python så klart det gikk

**Review by olros — APPROVED** [review]
Vakkert

**Pull request merged**